.buildkite/pipeline.yaml

```diff
@@ -6,7 +6,7 @@ steps:
       platform: "linux"
     timeout_in_minutes: 60
     env: &build-docker
-      DOCKER_IMAGE: "gcr.io/opensourcecoin/radicle-upstream:0.4.0"
+      DOCKER_IMAGE: "gcr.io/opensourcecoin/radicle-upstream"
```

Comment:
This won't work as we don't symlink `latest` in registry.

Reply:
Ah. I'm trying to just figure out what's going on here because `docker: Error response from daemon: error while mounting volume '': File exists (os error 17).` is not enlightening :smile:

Do you think it's worth just reverting back to doing it all in one script? :/

Reply:
It seems something is contending on certain state on disk, we should find out what that is and isolate it per step. Looked into other repos with parallel execution but couldn't find anything obvious.

Reply:
Ya, fair. I don't want this work to suck up my cycle time though. I'm going to put the calls back and create an issue for follow up work on this.

Reply:
https://github.com/radicle-dev/radicle-upstream/issues/500